modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/RuleList/RuleList.es.js

```diff
@@ -372,98 +364,265 @@ class RuleList extends Component {
+        </div>
+    )
+}
+
+/**
+ * Prints the DDM form card rule.
+ */
+const Label = ({content}) => {
+    return (
+        <span className="label label-lg label-secondary" data-original-title={content} title={content}>
+            <span className="text-truncate-inline">
+                <span className="text-truncate">{content}</span>
+            </span>
+	    </span>
+    )
+}
+
+
+/**
+ * Prints Rules Conditions.
+ */
+const Condition = ({
+    operandType,
+    operandLabel,
+    operandValue
+}) => {
+    const SubCondition = ({operandType}) => {
+        if(['double', 'integer', 'json', 'option', 'string', 'text', 'numeric'].includes(operandType)){
+            return  (<span>{Liferay.Language.get('value')} </span>);
+        }
+    
+        if(operandType === 'user'){
+            return (<span>{Liferay.Language.get('user')} </span>);
+        }
+
+        if(operandType === 'field'){
+            return (<span>{Liferay.Language.get('field')} </span>);
+        }
+
+        if(operandType === 'list'){
+            return (<span>{Liferay.Language.get('list')} </span>);
+        }
+
+        return null;
+    }
+
+    return (
+        <div className="ddm-condition-container">
+            <SubCondition operandType={operandType}/>
+
+            <Label content={operandLabel || operandValue} />
+	    </div>
+    );
+}
+
+/**
+ * Prints the show action.
+ */
+const Action = ({
+    action,
+    dataProviderName,
+    outputLabel,
+}) => {
+    if(action === 'auto-fill'){
+        return (
+            <>
+                {outputLabel.map(output => {
+                    <>
+                        <Label content={output} />
+                        {!isLast(output) ? ',' : ''}
+                    </>
+                })}
+                
+                <span>
+                    <b>
+                        {Liferay.Util.sub(
+                            Liferay.Language.get('autofill-x-from-data-provider-x'),
+                            dataProviderOutputFields,
+                            dataProviderName
+                        )}
+                    </b>
+                </span>
+            </>
+        );
+    }
+
+    if(action === 'calculate'){
+        return (
+            <>
+                <Label content={expression} />
+
+                <Label content={label} />
+
+                <span>
+                    <b>
+                        {Liferay.Util.sub(
+                            Liferay.Language.get('calculate-field-x-as-x'),
+                            expressionLabel,
+                            targetLabel
+                        )}
+                    </b>
+                </span>                
+            </>
+        );
+    }
+
+    const SubLabel = ({text, label}) => {
+        return (<>
+            <span>
+                <b>
+                    {text}
+                </b>
+            </span>
+
+            <Label content={label} />
+        </>);
+    }
+
+    if(action === 'enable'){
```

Comment:
```js
const ACTIONS = new Set(['enable', 'jump-to-page', 'require', 'show']);

if (ACTIONS.has(action)) {
    return <SubLabel text={Liferay.Language.get(action)} label={label} />
}

return null;
```

Reply:
Hi, @diegonvs I made that because of portal translation interpolation process.